src/arangodb/storage.py

```
@@ -792,42 +1097,149 @@ async def create_index(
+            if '/' in node_id:
+                collection, key = node_id.split('/', 1)
+                doc = self._client.get_document(collection, key)
+            else:
+                # Try common node collections
+                doc = None
+                for collection in ['nodes', 'nodes_file', 'nodes_chunk', 'nodes_concept']:
+                    try:
+                        doc = self._client.get_document(collection, node_id.replace('/', '_'))
+                        if doc:
+                            break
+                    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)